### PR TITLE
Style: always use "if" with braces, remove empty comments

### PR DESCRIPTION
**Pull request by proski**
This PR includes changes that I made manually in #51.

### CODE REVIEW

**Comment by proski** [comment]
Just an equivalent of "test this please". The PR checker got stuck in the "pending" state for days.

File: src/main/java/stashpullrequestbuilder/stashpullrequestbuilder/StashRepository.java [hunk 1]
```diff
@@ -243,8 +243,13 @@ private Boolean isPullRequestMergable(StashPullRequestResponseValue pullRequest)
       StashPullRequestMergableResponse mergable =
           client.getPullRequestMergeStatus(pullRequest.getId());
       boolean res = true;
-      if (trigger.isCheckMergeable()) res = res && mergable.getCanMerge();
-      if (trigger.isCheckNotConflicted()) res = res && !mergable.getConflicted();
+      if (trigger.isCheckMergeable()) {
+        res = res && mergable.getCanMerge();
```
Comment (jakub-bochenski) [inline src/main/java/stashpullrequestbuilder/stashpullrequestbuilder/StashRepository.java]: minor: maybe change to `&=` while we'r at it?

**Review by jakub-bochenski — APPROVED** [review]

File: src/main/java/stashpullrequestbuilder/stashpullrequestbuilder/StashRepository.java (hunk 1 above)
Comment (proski) [reply src/main/java/stashpullrequestbuilder/stashpullrequestbuilder/StashRepository.java]: Done. Also rebased. I like rebasing on top of my merged commits :smiley:

Comment (jakub-bochenski) [reply src/main/java/stashpullrequestbuilder/stashpullrequestbuilder/StashRepository.java]: I can only hope this happens more often :)

**Pull request merged**